static/styles/colors.css

```diff
@@ -0,0 +1,87 @@
+@theme {
+    /* Glitch */
+    --color-glitch-50: oklch(98.2% 0.002 95.7);  /* #f9f9f9 */
+    --color-glitch-100: oklch(95.9% 0.003 95.7); /* #f2f2f2 */
+    --color-glitch-200: oklch(91.8% 0.005 95.7); /* #e2e2e2 */
+    --color-glitch-300: oklch(86.6% 0.008 95.7); /* #cdcecd */
+    --color-glitch-400: oklch(79.9% 0.012 95.7); /* #afb1af */
+    --color-glitch-500: oklch(70.9% 0.017 95.7); /* #8c908c */
+    --color-glitch-600: oklch(57.7% 0.023 95.7); /* #727772 */
+    --color-glitch-700: oklch(48.2% 0.027 95.7); /* #5a605a */
+    --color-glitch-800: oklch(39.9% 0.031 95.7); /* #474c47 */
+    --color-glitch-900: oklch(27.1% 0.036 95.7); /* #2d312d */
+    --color-glitch-950: oklch(21.0% 0.037 95.7); /* #202320 */
+
+    /* Primary */
+    --color-primary-dark: oklch(0.987 0.024 297.388deg); /* #FAFBFA */
```

Comment:
Or I am getting confused and with primary-dark it is not refered to the primary color in dark mode and primary-light the primary color in light mode? Just the primary color a bit lighter or darker? In that case  I would prefer the use of tailwind scales from 100 to 900